students/Chay_Casso/Lesson01/generator_solution.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python3
+#
+# Generators
+# Chay Casso
+# 4/9/2018
+
+from itertools import *
```

Comment:
You don't want to get in the habit of doing `import *` as then people won't know where that function comes from and they might think its something else somewhere in the code.  Be explicit about the functions you are importing.